docs/cloud-native-security/cspm-faq.asciidoc

```diff
@@ -5,21 +5,13 @@
 == CSPM FAQ
 Frequently asked questions about the Cloud Security Posture Management (CSPM) integration and features.
 
-*Which cloud service providers are supported?*
-
-AWS is currently supported. Support for GCP & Azure is coming soon.
-
 *How often is my cloud security posture evaluated?*
 
 Cloud accounts are evaluated when you first deploy the CSPM integration and every 24 hours afterward.
 
-*Can I onboard multiple AWS accounts for posture monitoring?*
-
-Yes, you can monitor more than one cloud account. <<cspm-get-started, Add the CSPM integration>> again for each additional account you want to monitor.
-
-*Can I onboard multiple accounts via AWS organizations?*
+*Can I onboard multiple accounts at one time?*
 
-This capability is not currently supported but will be added in future iterations.
+Yes. Follow the onboarding instructions in the getting started guides for AWS, GCP, or Azure.
```

Comment:
Maybe consider linking to the guides from here? Same comment for the other update below.